recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
```

Comment:
since we have `remove_files_by_mask`

```suggestion


required_conan_version = ">=1.29.1"


```